README.md

```diff
@@ -2,27 +2,35 @@
 [![Build Status](https://travis-ci.org/projecthydra-labs/hydra-works.svg?branch=master)](https://travis-ci.org/projecthydra-labs/hydra-works)
 [![Coverage Status](https://coveralls.io/repos/projecthydra-labs/hydra-works/badge.svg?branch=master)](https://coveralls.io/r/projecthydra-labs/hydra-works?branch=master)
 
-Hydra implementation of a works model based on the [Portland Common Data Model](https://github.com/projecthydra-labs/hydra-pcdm) (PCDM).
+The Hydra::Works gem provides a set of [Portland Common Data Model](https://wiki.duraspace.org/display/FF/Portland+Common+Data+Model)-compliant models and associated behaviors around the broad concept of multi-file "works," the need for which was expressed by a variety of [community use cases](https://github.com/projecthydra-labs/hydra-works/tree/master/use-cases). The Hydra::Works domain model includes:
 
+ * **GenericFile**: a *pcdm:Object* that encapsulates one or more directly related *pcdm:File*s, such as a PDF document, its derivatives, and extracted full-text
+ * **GenericWork**: a *pcdm:Object* that holds zero or more **GenericFile**s and zero or more **GenericWork**s
+ * **Collection**: a *pcdm:Collection* that indirectly contains zero or more **GenericWork**s and zero or more **Collection**s
+
+View [a diagram of the domain model](https://docs.google.com/drawings/d/1-NkkRPpGpZGoTimEpYTaGM1uUPRaT0SamuWDITvtG_8/edit).
+
+Hydra::Works, which was inspired by Sufia, Worthwhile, and Curate, will provide interoperable models for use by a number of Hydra tools, including the aforementioned gems and more. Hydra::Works adoption need not be limited to the institutional/data repository use case; that is merely the starting point.
```

Comment:
I'm inclined to strike the whole paragraph. This seems like one of those instances where saying less is wiser and claerer than saying more.